docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,49 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
+
+- The `cy.log()`command will now display a line break where the `\n` character
+  is used. Addressed in
+  [#25467](https://github.com/cypress-io/cypress/pull/25467).
+- `.jsx`/`.tsx` file extensions will be utilized when generating a new empty
+  spec file inside component testing if the project contains at least one file
+  with those extensions. Addressed in
+  [#25318](https://github.com/cypress-io/cypress/pull/25318).
+- Added new `experimentalSkipDomainInjection` flag to disable Cypress from
+  setting `document.domain` on injection. Addressed in
+  [#25307](https://github.com/cypress-io/cypress/pull/25307).
+- Added new `experimentalMemoryManagement` flag to improve memory management in
+  Chromium-based browsers. Addressed in
+  [#25462](https://github.com/cypress-io/cypress/pull/25462).
+- Video output link in `cypress run` mode has been added to it's own line to
+  make the video output link more easily clickable in the terminal. Addressed in
+  [#23915](https://github.com/cypress-io/cypress/pull/23915).
+- Support `data-qa` selector in Selector Playground in addition to `data-cy`,
+  `data-test` and `data-testid`. Addressed in
+  [#25475](https://github.com/cypress-io/cypress/pull/25475).
+  - `.as()` now accepts an options argument, allowing test writers to store an
+    alias as a "query" or "static". Addressed in
+    [#25251](https://github.com/cypress-io/cypress/pull/25251).
+
+**Bugfixes:**
+
+- Fixes an issue where component tests could incorrectly treat new major
```

Comment:
This should read as past-tense
```suggestion
- Fixed an issue where component tests could incorrectly treat new major
```
We should also list which versions this was incorrectly reporting as supported instead of saying `certain dependencies`.

Reply:
The fix was for future-proofing the supported deps. As it stands, there are no versions currently released that we are saying we don't support (for example we said we supported Angular >=13, it's now been fixed to 13,14,15 and v16 is not released yet)

Reply:
past-tense added for Bugfixes in 4ba7d032b